### PR TITLE
Remove any migration logic from web logic

### PR DESCRIPTION
**Pull request by labkode**
Epic: EOSUSER retirement.

### CODE REVIEW

**Review by diocas — COMMENTED** [review]
Do you want to clean everything or just basic stuff? Cause there are still other mentions of old home around (i.e in the main.go and the `createHome`). Maybe the `createHome` would be safer to remove it.

File: api/storage_homemigration/storage_homemigration.go [hunk 1]
```diff
@@ -95,34 +95,11 @@ func (fs *eosStorage) getStorageForUser(ctx context.Context, u *api.User) (api.S
 	key := fmt.Sprintf("/eos/user/%s/%s", letter, u.AccountId)
```
Comment (diocas) [inline api/storage_homemigration/storage_homemigration.go:95]: this key is no longer used

File: api/storage_projectmigration/storage_projectmigration.go [hunk 2]
```diff
@@ -68,13 +68,6 @@ func (fs *eosStorage) getStorageForProject(ctx context.Context, fn string) (api.
 	key := fmt.Sprintf("/eos/project/%s/%s", letter, projectName)
```
Comment (diocas) [inline api/storage_projectmigration/storage_projectmigration.go:68]: same

File: api/storage_usermigration/storage_usermigration.go [hunk 3]
```diff
@@ -93,35 +93,12 @@ func (fs *eosStorage) getStorageForPath(ctx context.Context, letterPath string)
```
Comment (diocas) [inline api/storage_usermigration/storage_usermigration.go:93]: same

**Review by diocas — CHANGES_REQUESTED** [review]

**Comment by labkode** [comment]
> Do you want to clean everything or just basic stuff? Cause there are still other mentions of old home around (i.e in the main.go and the `createHome`). Maybe the `createHome` would be safer to remove it.

Only the minimum, whatever we do is going to be trashed in 6 months. The current changes should remove logic dependency on EOSUSER. The create home even if it enters through oldhome will create the homes in the new instances (harcoded in the homedir script).

**Comment by diocas** [comment]
That's true.. We now completely ignore the eos instance... So everything's fine.

**Review by diocas — APPROVED** [review]

**Pull request merged**